src/components/auth/Oauth/AuthorizationCode/fetchOAuthPopupURL.ts

```diff
@@ -1,5 +1,11 @@
 import { api, OauthConnectOperationRequest, ProviderApp } from 'services/api';
-import { getProviderName } from 'src/utils';
+import { getEnvVariable, getProviderName } from 'src/utils';
+
+const VITE_ENABLE_CSRF = getEnvVariable('VITE_AMP_ENABLE_CSRF', false);
+const NEXT_ENABLE_CSRF = getEnvVariable('NEXT_AMP_ENABLE_CSRF', false);
+const REACT_APP_ENABLE_CSRF = getEnvVariable('REACT_APP_ENABLE_CSRF', false);
```

Comment:
can you actually name this `REACT_APP_AMP_ENABLE_CSRF`? I know that's wordier but it's good to always add AMP to our env variables to minimize chances of conflict with other env variables builders have.